utils/verification_test.go

```diff
@@ -170,3 +171,20 @@ func TestPackageGetVerificationCodeFailsIfNilFileInSlice(t *testing.T) {
 		t.Fatalf("expected non-nil error, got nil")
 	}
 }
+func FuzzPackageCanGetVerificationCode(f *testing.F) {
+	f.Fuzz(func(t *testing.T, filename string, fileSPDXIdentifier string, checksums string) {
+		checks := []common.Checksum{}
+		for _, check := range strings.Fields(checksums) {
+			checks = append(checks, common.Checksum{Algorithm: common.SHA1, Value: check})
+		}
+		files := []*spdx.File{
+			{
+				FileName:           filename,
+				FileSPDXIdentifier: common.ElementID(fileSPDXIdentifier),
+				Checksums:          checks,
+			},
+		}
+
+		GetVerificationCode(files, "")
```

Comment:
This should at least check for errors, maybe something like: 

```suggestion
		_, err := GetVerificationCode(files, "")
		if err != nil {
			t.Fatal(err)
		}
```